webpack.common.js

```diff
@@ -36,9 +38,9 @@ module.exports = {
         use: [{
           loader: 'file-loader',
           options: {
-            name: '[name].[ext]',
+            name: '[name].[ext]?[hash]',
             outputPath: 'fonts/',
-            publicPath: '/_static/fonts',
+            publicPath: '../fonts/',
```

Comment:
I'm still a little confused by this. It feels like `publicPath` is applied in a backwards way. I feel it should describe how paths are altered for the `MiniCssExtractPlugin` instances for `css/theme.css`, not backwards, where we alter the `publicPath` for font files, assuming they are used in the `css/theme.css` file with the nested path.

I don't know. I think this is technically correct, it just logically feels backwards. :man_shrugging:

Reply:
That's okay, I'm more than happy for this change

Reply:
Aye, this seems to work. I think webpack just has me confused here.